examples/electron/src-gen/backend/main.js

```diff
@@ -29,15 +29,16 @@ function start() {
     application.use(express.static(path.join(__dirname, '../../lib'), {
         index: 'index.html'
     }));
-    application.start();
+
+    module.exports.default = application.start(false, 0)
 }
 
 Promise.resolve()
```

Comment:
`module.exports.default = Promise.resolve().then(...` should be here, otherwise, with a bad timing, you will get an NPE from `electron-main.js`